accord-core/src/main/java/accord/impl/InMemoryCommandStore.java

```diff
@@ -398,6 +398,13 @@ protected <T> void executeInContext(InMemoryCommandStore commandStore, PreLoadCo
         }
     }
 
+    @Override
+    public String toString() {
```

Comment:
I use the debugger a lot to make sure threading is correct, so this is for that use case